craft_parts/plugins/cmake_plugin.py

```diff
@@ -88,9 +88,7 @@ def get_build_packages(self) -> Set[str]:
 
     def get_build_environment(self) -> Dict[str, str]:
         """Return a dictionary with the environment to use in the build step."""
-        return {
-            "CMAKE_PREFIX_PATH": "{self._part_info.stage_dir}",
-        }
+        return {"CMAKE_PREFIX_PATH": f"{self._part_info.stage_dir}"}
```

Comment:
Side question, is this a better alternative?
```suggestion
        return {"CMAKE_PREFIX_PATH": str(self._part_info.stage_dir)}
```

Reply:
It is, using that instead.